### PR TITLE
geth: Introduced config for a log directory

### PR DESCRIPTION
**Pull request by hackaugusto**
`geth_create_blockchain` can be used for testing or for setting up test
environments, for test environments it make sense to redirect the geth
output to log files and not clutter the user's terminal.

### CODE REVIEW

**Comment by hackaugusto** [comment]
Usage example:

https://gist.github.com/hackaugusto/c522f12426d771d60e063516c3107ff3

**Comment by LefterisJP** [comment]
Rebased the branch on master.

File: raiden/tests/utils/blockchain.py [hunk 1]
```diff
@@ -209,7 +209,8 @@ def geth_create_blockchain(
         p2p_ports,
         base_datadir,
         verbosity,
-        genesis_path=None):
+        genesis_path=None,
```
Comment (LefterisJP) [inline raiden/tests/utils/blockchain.py]: Can you maybe add a docstring to this function into which you also provide the example usage gist you linked in the PR's comments themselves?

**Comment by LefterisJP** [comment]
Looks good to me, just two things.

1. A docstring containing the example usage gist could be added to `geth_create_blockchain`.
2. So the default value still remains no logging from what I see. Do we really want that? Personally I think I would find it better scouring through a file, than looking at my stdout.

**Pull request merged**